### DISCUSSION TITLE
Plugin Template for installing predefined profiles?

### DISCUSSION THREAD

**Discussion by Annorax10** [elgatosf/elgato#30] [Help]
Is there a plugin template available that only installs predefined Streamdeck profiles? I am trying to build something that only installs a set of profiles I've created, to make it easier for other streamers on my shared channel to set up my pre-built profile setups without having to relink every file path.

**Comment by ChekTek** [comment]
Hi @Annorax10, 

In our new samples repo, the [Lights Out](https://github.com/elgatosf/streamdeck-plugin-samples/tree/main/lights-out/com.elgato.lightsout.sdPlugin) plugin is a good example of bundling a profile inside of a plugin. You can also references the [profile docs](https://docs.elgato.com/streamdeck/sdk/guides/profiles/).
